src/utils/selectors.js

```diff
@@ -1,18 +1,14 @@
 // @flow
-import { flatten, map, pipe, prop, uniq } from 'ramda'
+import { flatten, map, prop, uniq } from 'ramda'
 
 interface IEvent {
   date: Date;
 }
 
 const viewTags = talk => (talk.tags ? talk.tags : [])
 
-export let eventTags = pipe(
-  prop('talks'),
-  map(viewTags),
-  flatten,
-  uniq,
-)
+export let eventTags = e =>
+  e |> prop('talks') |> map(viewTags) |> flatten |> uniq
```

Comment:
In my opinion, this repetition `e => e |>...` looks a bit strange, after pretty descriptive curried `.pipe` :)

Reply:
agree

Reply:
That is because rambda's pipe and native pipeline operator do different things, but use same name. So e.g. in Elm you have two options: composition using << operatior and create pipelines using |> operator. Through my humble experience in elm, I learn that it is better use pipeline operator when we need the value now (e.g. in render function in react), because it doesn't support delayed evaluation. In this case it reads super awesome.
As for composition operator - I think it is clear, that it should be used for creating helper functions as in your example.

Reply:
Also in ReasonML doesn't support composition, and only pipelines. JFYI.

Feels unnatural from the very beginning. But used to it almost instantly